CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
+## v0.1.0 [2023-05-25]
+
+_Bug fixes_
+
+- Fixed `followers` and `following` columns in `mastodon_toot_favourite` table to be of `bigint` datatype. ([#16](https://github.com/turbot/steampipe-plugin-mastodon/pull/16))
```

Comment:
```suggestion
- Fixed the `followers` and the `following` columns in `mastodon_toot_favourite` table to be of `INT` datatype instead of `JSON`. ([#16](https://github.com/turbot/steampipe-plugin-mastodon/pull/16))
```